src/v/cluster/cluster_utils.cc

```diff
@@ -109,7 +109,30 @@ ss::future<> maybe_create_tcp_client(
                       std::optional<ss::tls::credentials_builder> credentials) {
                   if (credentials) {
                       return credentials
-                        ->build_reloadable_certificate_credentials();
+                        ->build_reloadable_certificate_credentials(
+                          [](
+                            std::unordered_set<ss::sstring> const& updated,
```

Comment:
let's use `const type&` just to stay consistent with the rest of the code base. i think there is another instance of this in one of the previous commits.